SOEM/CMakeLists.txt

```diff
@@ -15,9 +15,9 @@ if(WIN32)
   set(OS "win32")
   include_directories(oshw/win32/wpcap/Include)
   if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-    link_directories(${CMAKE_SOURCE_DIR}/oshw/win32/wpcap/Lib/x64)
+    link_directories(oshw/win32/wpcap/Lib/x64)
   elseif(CMAKE_SIZEOF_VOID_P EQUAL 4)
-    link_directories(${CMAKE_SOURCE_DIR}/oshw/win32/wpcap/Lib)
+    link_directories(oshw/win32/wpcap/Lib)
```

Comment:
I see why this is required. Would an upstream contribution to the original SOEM code make sense as well? Then we wouldn't have to make sure we keep this change to original SOEM code with every sync...

Reply:
Agreed. A pull request to the upstream is created.

Reply:
great thanks!

Reply:
@mgruhler Happy new year! And just wanted to share some good news that the changes made it way to the upstream. What do you think if I pull the `SOEM` from the latest master branch?

Reply:
@seanyen happy new year to you too.
Just had a look at this yesterday, prior to the merge. Great to see that this made it upstream.

I'm hesitant to pull in any non-release versions of upstream, though. If there is no critical bugfix, I'd like to track releases only.

But I'd be fine with merging your changes as they are now (given that the tests that I'll perform this week (big promise :grinning:) turn out fine) and reverting the changes to this file prior to the next sync with upstream.
What do you think about this approach?

Reply:
@mgruhler That works too. Sounds like a good plan to me.

Reply:
tracked in #34